README.md

```diff
@@ -2,20 +2,18 @@
 
 ### How Does it Work?
 
-[Apidocs.co](https://apidocs.co) uses Github Code Search API against this repo to provide Code Samples directly within pages.
+[Apidocs.co](https://apidocs.co) uses the GitHub Code Search API against this repo to provide Code Samples directly within pages.
 
-Because Github Code Search API is limited to single user or repo, this repository aggregates multiple relevant repos so they can all be searchable in a single request.
+Because the GitHub Code Search API is limited to a single user or repo, this repository aggregates multiple relevant repos so they can all be searchable in a single request.
 
 ### Limitations
 
-* It's plain text search - some generic names like `Application` can have lot's of false positives.
-* It's limited to certain entity types (eg `Class`, `Method`, `Property`, etc)
+* It's plain text search &ndash; some generic names like `Application` can trigger many false positives
```

Comment:
What's the reason for the escaped dash in github markup?

Reply:
typographical esthetics; compare these two here: 
- a hyphen, or minus character
&ndash; an N dash

Reply:
I always something new to learn 😊
Thanks